src/types.ts

```diff
@@ -276,6 +276,8 @@ export interface CustomScoringConfig {
   key_name: string;
 }
 
+export const ScoringErrorValue = -1;
```

Comment:
What happens if they choose to use -1 as a score?

Reply:
Fixed after commit https://github.com/HammingHQ/evals-ts/pull/46/commits/4deae898c4676c00fd49e91f2cf34b7c2a47ce79 using the prefix method as discussed.
<img width="1298" alt="image" src="https://github.com/HammingHQ/evals-ts/assets/55324813/e295cda5-fff0-4494-a8da-aa8dd795dfaf">